docker/docker-compose.yml

```diff
@@ -279,13 +291,20 @@ services:
       interval: 1s
       timeout: 5s
       retries: 5
-  redpanda:
+  connector-node:
+    # TODO: use nightly build to keep aligned with other components
+    image: ghcr.io/risingwavelabs/risingwave-connector-node:latest
```

Comment:
`ghcr.io/risingwavelabs/risingwave:latest` refers to the latest nightly build, but  
`ghcr.io/risingwavelabs/risingwave-connector-node` doesn't have nightly build yet. This might be a problem, because they may be incompatiable with each other. cc. @huangjw806

Reply:
With https://github.com/risingwavelabs/risingwave/pull/8414 we may include the connector-node in the RW nightly build image.

Reply:
The previous nightly build contained the v0.1.17 version of the connector node, which is nightly after https://github.com/risingwavelabs/risingwave/pull/8414  is merged.